motor/providers/container/container.go

```diff
@@ -110,17 +121,27 @@ func NewDockerEngineImage(endpoint *providers.Config) (ContainerProvider, error)
 		return nil, err
 	}
 
-	log.Debug().Msg("found docker engine image " + ii.ID)
-	img, rc, err := image.LoadImageFromDockerEngine(ii.ID)
+	labelImageId := ii.ID
+	splitLabels := strings.Split(ii.Labels["docker.io/digests"], ",")
+	if len(splitLabels) > 1 {
+		labelImageIdFull := splitLabels[0]
+		splitFullLabel := strings.Split(labelImageIdFull, "@")
+		if len(splitFullLabel) > 1 {
+			labelImageId = strings.Split(labelImageIdFull, "@")[1]
+		}
+	}
+
+	// This is the image id that is used to pull the image from the registry.
+	log.Debug().Msg("found docker engine image " + labelImageId)
+	if ii.Size > 1024 && !disableInmemoryCache { // > 1GB
+		log.Warn().Int64("size", ii.Size).Msg("The image is bigger than 1GB, this will require a lot of memory. Consider disabling the in-memory cache with `--disable-cache=true`.")
```

Comment:
```suggestion
		log.Warn().Int64("size", ii.Size).Msg("The image is bigger than 1GB, this will require a lot of memory. Consider disabling the in-memory cache with `--disable-cache=true`.")
```

The message reads a bit weird. Not sure how to re-formulate it to be better though

Reply:
@misterpantz Do you have a suggestion?

Reply:
How about...
```suggestion
		log.Warn().Int64("size", ii.Size).Msg("Because the image is larger than 1 GB, the scan will require a lot of memory. Consider disabling the in-memory cache by adding this flag to the command: `--disable-cache=true`")
```

Reply:
Thanks @misterpantz